### PR TITLE
[FOLLOWUP] [#1401] Fix(dashboard): Fix the display of app reg time on app page

### PR DESCRIPTION
**Pull request by maobaolong**
### What changes were proposed in this pull request?

Display app reg time in dashboard.

### Why are the changes needed?

Fix typo error introduced by #1868 

### Does this PR introduce _any_ user-facing change?

User can see app reg time from dashboard.

### How was this patch tested?

Open a dashboard ui and submit an application and check the app reg time.

### CODE REVIEW

**Comment by maobaolong** [comment]
@yl09099 Hi, Would you like to take a look at this PR? Thanks.

**Comment by yl09099** [comment]
> @yl09099 Hi, Would you like to take a look at this PR? Thanks.

You should add a space, otherwise you can't sort. eg: el-table-column sortable

File: dashboard/src/main/webapp/src/pages/ApplicationPage.vue [hunk 1]
```diff
@@ -55,7 +55,7 @@
       >
         <el-table-column prop="appId" label="AppId" min-width="180" sortable fixed />
         <el-table-column prop="userName" label="UserName" min-width="180" sortable />
-        <el-table-columnsortable
+        <el-table-column
           prop="registrationTime"
           label="Registration Time"
           min-width="180"
```
Comment (maobaolong) [inline dashboard/src/main/webapp/src/pages/ApplicationPage.vue:61]: The sortable is in L63

https://github.com/apache/incubator-uniffle/pull/1963/files#diff-f9bf9c7f3ab29a053cbc120fd3db975dce2e1f52b4e4295b90e08aaa840930c9R63

**Comment by maobaolong** [comment]
> > @yl09099 Hi, Would you like to take a look at this PR? Thanks.
> 
> You should add a space, otherwise you can't sort. eg: el-table-column sortable

@yl09099 the sortable is over here

image

File: dashboard/src/main/webapp/src/pages/ApplicationPage.vue (hunk 1 above)
Comment (yl09099) [reply dashboard/src/main/webapp/src/pages/ApplicationPage.vue:61]: > The sortable is in L63
> 
> https://github.com/apache/incubator-uniffle/pull/1963/files#diff-f9bf9c7f3ab29a053cbc120fd3db975dce2e1f52b4e4295b90e08aaa840930c9R63

OK. LGTM!

**Review by rickyma — APPROVED** [review]

**Comment by rickyma** [comment]
Merged to master. Thanks @maobaolong @yl09099

**Comment by maobaolong** [comment]
@rickyma @yl09099 Thanks for your review！

**Pull request merged**